docs/release/trg-1/trg-1-1.md

```diff
@@ -2,21 +2,37 @@
 title: TRG 1.01 - README.md
 ---
 
-| Status | Created      | Post-History |
-|--------|--------------|--------------|
-| Active | 07-Mar-2023  |              |
-| Draft  | 13-Sept-2022 |              |
+| Status | Created      | Post-History                             |
+|--------|--------------|------------------------------------------|
+| Update | 06-Dec-2023  | add detailed information for README file |
+| Active | 07-Mar-2023  |                                          |
+| Draft  | 13-Sept-2022 |                                          |
 
 ## Why
 
-A good `README.md` file is the starting point for everyone opening a repository. It should help them find all critical information in an easy way.
+A good `README.md` file is the starting point for everyone opening your repository. It should help them find all critical information in an easy way.
+A well-maintained `README.md` shows the high level of professionalism of the project. It demonstrates that the developers care about user experience and want to make it easy for others to understand and use their work.
 
 ## Description
 
 Each repository **must** contain a `README.md` file which **shall** cover following topics:
 
-- Basic description of repository and its content
-- Installation instructions to get component working
-- If required, additional post installation configuration steps to finish installation
+- Basic documentation of the repository, its content and its structure
+- Installation instructions
+  - If required necessary step(s) to finish the installation
+- Usage examples and intended use of your product
+- License information
+- Troubleshooting and FAQs to help users solve common problems
+- Reference your Contribution guideline in the `CONTRIBUTING.md` file
+- Contact information to help users getting in touch with you
+- Badges to show the status of your project (e.g. build status, code coverage, [etc](https://github.com/badges/shields/blob/master/README.md).)
 
-If installation instructions are too comprehensive for `README.md` move content to `INSTALL.md` file, see [TRG 1.2](trg-1-2.md).
+Best practices:
+
+- If installation instructions for your product are too complex being part of `README.md` move its content to a separate `INSTALL.md` file, see [TRG 1.02](trg-1-2.md).
+- If your project is too complex to be described in a single `README.md` file, consider creating a `docs` folder and link your `README.md` to it.
```

Comment:
Not sure if that statement doesn't bring a little confusion which might be percived as 'docs' being optional, however 'docs' is mandatory anyway despite of having complex or not README. 
Would maybe reword it slightly like "it may be beneficial to relocate specific sections to a dedicated docs/ folder"

Reply:
[cde0086](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/cde008647f7011db8e47929d62c096456082e57a)